CHANGELOG.md

```diff
@@ -1,4 +1,43 @@
-## 1.10.0 (Unreleased)
+## 2.0.0 (Unreleased)
+
+NOTES:
+* `datadog_monitor`: The silence attribute is beginning its deprecation process, please use `datadog_downtime` instead ([#221](https://github.com/terraform-providers/terraform-provider-datadog/issues/221))
+
+IMPROVEMENTS:
+* `datadog_monitor`: Use ForceNew when changing the Monitor type ([#236](https://github.com/terraform-providers/terraform-provider-datadog/issues/236))
+* Update the vendored go client to `v2.21.0`. ([#230](https://github.com/terraform-providers/terraform-provider-datadog/issues/230))
```

Comment:
I'd propose moving this to bottom, as it's something that's transparent to users of the provider (least important from my POV)